demo-app/src/lib/mysql-uqi-client.ts

```diff
@@ -103,13 +103,22 @@ async function createMysqlUqiClient(config: MysqlConfig): Promise<UqiClient> {
     return asyncGenerator()
   }
 
+  let user = parsedUrl.username
```

Comment:
What do you think about extracting this into a function? `extractUser(url: URL): string`?